src/test/java/org/folio/search/controller/SearchAuthorityIT.java

```diff
@@ -90,7 +90,12 @@ void searchByAuthorities_parameterized_all(String query, String value) throws Ex
       arguments("personalName all {value}", "\"Gary A. Wills\""),
       arguments("personalName all {value}", "gary"),
       arguments("personalName == {value}", "\"gary a.*\""),
-      arguments("personalName == {value} and headingType==\"Personal Name\"", "\"gary a.*\"")
+      arguments("personalName == {value} and headingType==\"Personal Name\"", "\"gary a.*\""),
+      arguments("sftPersonalName = {value}", "\"stf personal name\""),
+      arguments("sftPersonalName == {value}", "\"stf personal name\""),
```

Comment:
We should change value on authority sample too
```suggestion
      arguments("sftPersonalName == {value}", "\"sft personal name\""),
```